src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -171,71 +199,75 @@ const NewQuestionnaireScreen = () => {
-              </Typography>
-              <Typography variant="h6" gutterBottom sx={{ mb: 1, mt: 1 }}>
-                {strings.newQuestionnaireScreen.requiredAnswers} {questionnaire.passScore}
-              </Typography>
-              <Slider
-                value={questionnaire.passScore}
-                onChange={handlePassScoreSliderChange}
-                step={1}
-                marks
-                min={0}
-                max={countCorrectAnswers()}
-                valueLabelDisplay="auto"
-                sx={{ mt: 1, width: "70%" }}
-              />
-            </Box>
-            <Button
-              sx={{ display: "flex", alignItems: "center", mt: 8 }}
-              id="save-submit"
-              size="large"
-              variant="contained"
-              color="success"
-              onClick={saveQuestionnaire}
-              disabled={loading}
+            <CardActions
+              sx={{
+                display: "flex",
+                justifyContent: "flex-end",
+                padding: 0,
+                alignItems: "flex-start",
+                flexDirection: { xs: "column", sm: "row" },
+                width: "100%"
+              }}
             >
-              {loading ? (
-                <CircularProgress size={24} color="inherit" />
-              ) : (
-                strings.newQuestionnaireScreen.saveButton
-              )}
-            </Button>
-          </CardActions>
+              <Box sx={{ display: "flex", flexDirection: "column", width: "70%", mr: 4 }}>
+                <Typography
+                  variant="h6"
+                  gutterBottom
+                  sx={{ display: "flex", alignItems: "center", mb: 1, mt: 1 }}
+                >
+                  {strings.newQuestionnaireScreen.countedAnswers} {countCorrectAnswers()}
+                </Typography>
+                <Typography variant="h6" gutterBottom sx={{ mb: 1, mt: 1 }}>
+                  {strings.newQuestionnaireScreen.requiredAnswers} {questionnaire.passScore}
+                </Typography>
+                <Slider
+                  value={questionnaire.passScore}
+                  onChange={handlePassScoreSliderChange}
+                  step={1}
+                  marks
+                  min={0}
+                  max={countCorrectAnswers()}
+                  valueLabelDisplay="auto"
+                  sx={{ mt: 1, mb: 1, width: "70%" }}
+                />
+              </Box>
+              <Tooltip title={renderUpdatedTooltips()} placement="bottom">
+                <span>
+                  <Button
+                    sx={{ display: "flex", alignItems: "center", mt: 6, mr: 4 }}
+                    id="save-submit"
+                    size="large"
+                    variant="contained"
+                    color="success"
+                    onClick={saveQuestionnaire}
+                    disabled={loading || !questionnaire.title || !questionnaire.description}
```

Comment:
Just pointing out these conditions could be placed into a named variable e.g. isDisabled just before the main component render. Not strictly necessary in this case but would be a good idea if the condition list grew any larger.